procedures: edit the sections about the che-editor.yaml file (#2466)
codenvy/che-docs: modules/end-user-guide/partials/proc_setting-up-che-editor-yaml.adoc

[id="setting-up-che-editor-yaml_{context}"]
= Setting up che-editor.yaml

> To define the same in-browser IDE for all workspaces that will clone the same remote Git repository of your project, you can use the `che-editor.yaml` file. Using this file allows you to set a common default editor for your team and to provide new contributors with a default editor that is most suitable for your project. You can also use the `che-editor.yaml` file when you need to set a different IDE default for a particular repository of your project rather than use the default IDE of your organization's {prod-short} instance.

Reviewer: ```suggestion
To define the same in-browser IDE for all workspaces that will clone the same remote Git repository of your project, you can use the `che-editor.yaml` file.

This way, you can set a common default editor for your team and provide new contributors with the most suitable editor for your project. You can also use the `che-editor.yaml` file when you need to set a different IDE default for a particular project repository rather than the default IDE of your organization's {prod-short} instance.
```

Author: Thank you, Rolfe. I'm applying your suggestion. I just moved the second sentence to the second paragraph.

[id="setting-up-che-editor-yaml_{context}"]
= Setting up che-editor.yaml

> To define the same in-browser IDE for all workspaces that will clone the same remote Git repository of your project, you can use the `che-editor.yaml` file. Using this file allows you to set a common default editor for your team and to provide new contributors with a default editor that is most suitable for your project.

Author: @l0rd, WDYT?
```suggestion
To define the same in-browser IDE for all workspaces that will clone the same remote Git repository of your project, you can use the `che-editor.yaml` file. Using this file allows you to set a common default editor for your team and to provide new contributors with a default editor that is most suitable for your project. You can also use the `che-editor.yaml` file when you need to set a different IDE default for a particular repository of your project rather than use the default IDE of your organization's {prod-short} instance.
```

Reviewer 2: 'when you need set' ==> 'when you need to set'. Beyond that 👍

> To define the same in-browser IDE for all workspaces that will clone the same remote Git repository of your project, you can use the `che-editor.yaml` file. This is convenient when, as a developer using {prod-short}, you clone your personal fork of the project repository into your {prod-short} user's workspaces, and this way you don't have to specify your IDE every time you start another workspace with a clone of that repository.

Reviewer 2: > This is convenient when, as a developer using {prod-short}, you clone your personal fork of the project repository into your {prod-short} user's workspaces, and this way you don't have to specify your IDE every time you start another workspace with a clone of that repository.

The main reason you do that is because you want contributors to your project to have a different default IDE 
(compared to the default IDE of the Che instance). 

Doing it on a fork, but not on the origin repository, is problematic if you then are going to open PRs. And bookmarking a URL with the `che-editor` parameter is easier than creating and pushing a `che-editor.yaml` file in your fork. So no, you are doing to share it with the rest of the team, in particular for new contributors that may not know what's the best IDE for your project.

> To define the same in-browser IDE for all workspaces that will clone the same remote Git repository of your project, you can use the `che-editor.yaml` file. This is convenient when, as a developer using {prod-short}, you clone your personal fork of the project repository into your {prod-short} user's workspaces, and this way you don't have to specify your IDE every time you start another workspace with a clone of that repository.

Reviewer 3: ```suggestion
To define the same in-browser IDE for all workspaces that will clone the same remote Git repository of your project, use the `che-editor.yaml` file. This is convenient for developers using {prod-short} who clone their personal fork of the project repository into their {prod-short} user's workspaces. This way you don't have to specify your IDE every time you start another workspace with a clone of that repository.
```

Author: Thank you, Jana. This appears to be outdated, so I'm resolving this conversation.

> To define the same in-browser IDE for all workspaces that will clone the same remote Git repository of your project, you can use the `che-editor.yaml` file. This is convenient when, as a developer using {prod-short}, you clone your personal fork of the project repository into your {prod-short} user's workspaces, and this way you don't have to specify your IDE every time you start another workspace with a clone of that repository.

Author: @l0rd, great, thank you for correcting me. WDYT? 
```suggestion
To define the same in-browser IDE for all workspaces that will clone the same remote Git repository of your project, you can use the `che-editor.yaml` file. Using this file allows you to set a common default editor for your team and to provide new contributors with a default editor that is most suitable for your project.
```

Reviewer 2: 👍

.Prerequisites

> * The IDE `id` from the table in xref:selecting-a-workspace-ide.adoc[].

Reviewer 2: ```suggestion
* The IDE `id` from the table in xref:selecting-a-workspace-ide.adoc[]. Note that it's possible to use the `reference` to a remote IDE definition or an `inline` definition instead of the `id`. This is detailed in the options sections below.
```

Author: Thank you for mentioning this, Mario. In response to it, I've completely rewritten both sections, so this suggestion is now outdated.
